src/pages/index.js

```diff
@@ -11,7 +11,7 @@ const Home = ({
 }) => {
   const FAQs = edges
     .filter(edge => !!edge.node.frontmatter.question)
-    .map(edge => <Grid item xs={12}><FAQCard faq={edge.node}/></Grid>)
+    .map(edge => <Grid key={edge} item xs={12}><FAQCard key={edge} faq={edge.node}/></Grid>)
```

Comment:
Instead of using the object as the key, best to use the object's id (edge.node.id) for good practice.